### PR TITLE
Close #111 (SDF file ➔ SDF document)

### LINKED ISSUE #111: (editorial): Find term for "SDF file" (CH-20, CH-21)
Can we call it an "SDF model"?

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: sdf.md [hunk 1]
```diff
@@ -642,18 +642,22 @@ All names in SDF are case-sensitive.
...
 
-SDF definitions are contained in SDF files.  One or more SDF files can
-work together to provide the definitions and declarations that are the
-payload of the SDF format.
+SDF definitions are contained in SDF documents, together with data
+about the SDF document itself (information block).
+Definitions and declarations from additional SDF documents can be
+referenced; together with the definitions and declarations in the
+referencing SDF document they build the model expressed by the
+referencing SDF document.
```
Comment (JKRhb) [inline sdf.md:650]: As there are a number of instances of "SDF model" left in the draft, should it maybe be introduced as its own term in the Terminology section and also be mentioned here?

File: sdf.md [hunk 2]
```diff
@@ -1399,13 +1405,20 @@ The requirements for high level composition include the following:
 
 ## Paths in the model namespaces
 
-The model namespace is organized according to terms that are defined in the definition files that are present in the namespace. For example, definitions that originate from an organization or vendor are expected to be in a namespace that is specific to that organization or vendor. There is expected to be an SDF namespace for common SDF definitions used in OneDM.
+The model namespace is organized according to terms that are defined
+in the SDF documents that are present in the namespace. For example, definitions that originate from an organization or vendor are expected to be in a namespace that is specific to that organization or vendor. There is expected to be an SDF namespace for common SDF definitions used in OneDM.
```
Comment (JKRhb) [inline sdf.md:1409]: Not the main point of this PR, but besides in this sentence, OneDM is only mentioned in an example and the abbreviated title. Should the term be explained here? Or the sentence removed?

**Review by JKRhb — COMMENTED** [review]
Looks good to me in general, see two comments below

File: sdf.md [hunk 3]
```diff
@@ -642,18 +642,22 @@ All names in SDF are case-sensitive.
 
 # SDF structure
 
-SDF definitions are contained in SDF files.  One or more SDF files can
-work together to provide the definitions and declarations that are the
-payload of the SDF format.
+SDF definitions are contained in SDF documents, together with data
+about the SDF document itself (information block).
+Definitions and declarations from additional SDF documents can be
```
Comment (akeranen) [inline sdf.md:647]: This sentence ended up a bit complex. Would it be simpler to say some thing like "Using references an SDF document can use definitions and declarations from other SDF documents to build models.".

File: sdf.md (hunk 2 above)
Comment (akeranen) [reply sdf.md:1409]: Indeed, probably the OneDM note is not needed here.

Comment (akeranen) [reply sdf.md:1409]: Also now realised "that are present in the namespace" is a bit confusing here. Would something like "that contribute to the namespace" be more clear?

**Review by akeranen — APPROVED** [review]

File: sdf.md (hunk 3 above)
Comment (cabo) [reply sdf.md:659]: Yes, but models are also built when other documents are not referenced (the point here is how models get to be).
I have simplified the sentence slightly without losing this point...

File: sdf.md (hunk 1 above)
Comment (cabo) [reply sdf.md]: I created a new issue
https://github.com/ietf-wg-asdf/SDF/issues/128

The "model" is mentioned in line 649, do you want to change this to "SDF model"?

**Pull request merged**